server.c

```diff
@@ -201,7 +206,18 @@ int run_server(const char *port, const char *cert, const char *key)
         return 1;
     }
 
-    printf("starting server on port %s\n", port);
+    uint64_t pid;
+
+    #ifdef __APPLE__
+        pthread_threadid_np(NULL, &pid);
+    #else
+        pid = syscall(SYS_gettid);
+    #endif
```

Comment:
We should wrap this in a `uint64_t get_current_pid()` function.

Reply:
Done in https://github.com/rbruenig/qperf/pull/5/commits/221035e4208987ca782bf32bdfcf72a3311b8a00 as part of `common`